### ISSUE TITLE
Build Status icons may benefit from contrast improvement

### ISSUE THREAD

**Issue by eplodn** [jenkinsci/pipeline-graph-view-plugin#281]
### What feature do you want to see added?

We're testing Graph View next to Blue Ocean, and the build status icons of BlueOcean have better contrast in my opinion.

1. "Not built" — we have really, really big graphs where many stages are "not built". In BlueOcean, "Not Built" and "Skipped" are represented similarly — having empty gray circles so it's not getting in your face while trying to understand what's wrong. In Graph View, "Skipped" are circles which are empty (good) but black (gray might be better). "Not built" are represented by blue circles with three dots in them, as if trying to hint "there is some additional info" (there isn't). The blue color stands out to imply some essence where there is almost none.
2. "Success" — white circle with green outline and green "check". Would be better if it was a green circle with a white check.
3. "Failure" — white circle with red outline and red "x", which frankly doesn't look that different from the other stages (either successful or skipped/not built). Considering that most of the time we are only looking to understand why the pipeline has _failed_, the failure stage is the most important for us. It would make much sense to make the icon red and bold so it would have a greater contrast from the other stage status icons.

In short, the skipped/not built stages have near-zero information, successful stages have some information, but the failed stages have the most information. It would make sense to visually adjust the icons to better direct the attention to the stages having the most relevant information.

Thanks!

**Comment by timja** [comment]
I think resolved now, the icons have been changed:

<img width="705" alt="Image" src="https://github.com/user-attachments/assets/bae1f837-cdbf-4f3b-bc60-888f343dbb3a" />
